app/src/main/java/com/academy/ramon/marvelcomicviewer/components/RetrofitModule.java

```diff
@@ -0,0 +1,36 @@
+package com.academy.ramon.marvelcomicviewer.components;
+
+import com.academy.ramon.marvelcomicviewer.api.MarvelAPI;
+
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by Ramon on 1/3/2017.
+ */
+
+@Module
+public class RetrofitModule {
+
+    @Provides
+    @Singleton
+    Retrofit provideRetrofit() {
+        return new Retrofit.Builder()
+                .addCallAdapterFactory(RxJavaCallAdapterFactory.create())
+                .addConverterFactory(GsonConverterFactory.create())
+                .baseUrl(MarvelAPI.ENDPOINT)
```

Comment:
I dont quite get how to do this since im not calling this method any more, would it be like:
```java
@Module
public interface RetrofitModule {

    @Provides
    @Singleton
    Retrofit provideRetrofit() 
```

then like: 
``` java
@Module
public class MarvelRetrofitModule implements RetrofitModule {

    @Provides
    @Singleton
    Retrofit provideRetrofit() {
        return new Retrofit.Builder()
                .addCallAdapterFactory(RxJavaCallAdapterFactory.create())
                .addConverterFactory(GsonConverterFactory.create())
                .baseUrl(MarvelAPI.ENDPOINT)
                .build();
    }
```